vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -637,6 +738,54 @@ unit:BAR
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Bar" ;
 .
+unit:BAR-L-PER-SEC
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e+02 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "product of the unit bar and the unit litre divided by the SI base unit second" ;
+  qudt:hasQuantityKind quantitykind:Power ;
+  qudt:iec61360Code "0112/2///62720#UAA326" ;
+  qudt:uneceCommonCode "F91" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "BAR L PER SEC" ;
+  skos:prefLabel "bar litre per second" ;
+.
+unit:BAR-M3-PER-SEC
+  a qudt:Unit ;
+  qudt:conversionMultiplier 1e+02 ;
+  qudt:conversionOffset 0e+00 ;
+  qudt:description "product out of the 0,001-fold of the unit bar and the power of the SI base unit metre with the exponent 3 divided by the SI base unit second" ;
+  qudt:hasQuantityKind quantitykind:Power ;
+  qudt:iec61360Code "0112/2///62720#UAA814" ;
+  qudt:uneceCommonCode "F96" ;
+  rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
+  rdfs:label "BAR M3 PER SEC" ;
+  skos:prefLabel "bar metre cubed per second" ;
+.
+unit:BAR-PER-BAR
```

Comment:
If this is something i have added it is because it is part of the IEC unit vocabulary, so we have to decide whether we want to be compliant or not. If yes, then no discussion is necessary. If not, then we need to tell siemens why not (and probably a lot more interested parties).